# Housing Market

### Introduction:

This time we will create our own dataset with fictional numbers to describe a house market. As we are going to create random data don't try to reason of the numbers.

### Step 1. Import the necessary libraries

In [1]:
import pandas as pd
import numpy as np

### Step 2. Create 3 differents Series, each of length 100, as follows: 
1. The first a random number from 1 to 4 
2. The second a random number from 1 to 3
3. The third a random number from 10,000 to 30,000

In [5]:
fst = pd.Series(np.random.randint(low=1,high=5,size=100))
snd = pd.Series( np.random.randint(low=1, high=4, size=100))
thd = pd.Series(np.random.randint(low=10000,high=30001,size=100))


### Step 3. Let's create a DataFrame by joinning the Series by column

In [8]:
mkt = pd.concat([fst,snd,thd], axis=1)
mkt.head()

0  1      2
0  4  2  18186
1  4  2  20787
2  1  1  29403
3  3  3  21087
4  3  1  10184

### Step 4. Change the name of the columns to bedrs, bathrs, price_sqr_meter

In [14]:
mkt.rename(columns={0:'bedrs', 1:'bathrs', 2:'price_sqr_meter'}, inplace=True)
mkt

bedrs  bathrs  price_sqr_meter
0       4       2            18186
1       4       2            20787
2       1       1            29403
3       3       3            21087
4       3       1            10184
..    ...     ...              ...
95      3       2            18882
96      1       2            24236
97      4       2            16585
98      2       3            29032
99      3       1            24115

[100 rows x 3 columns]

### Step 5. Create a one column DataFrame with the values of the 3 Series and assign it to 'bigcolumn'

In [16]:
bigcolumn = pd.concat([fst,snd,thd], ignore_index=True)
bigcolumn.shape

(300,)

### Step 6. Oops, it seems it is going only until index 99. Is it true?

In [17]:
bigcolumn.tail()

295    18882
296    24236
297    16585
298    29032
299    24115
dtype: int64

### Step 7. Reindex the DataFrame so it goes from 0 to 299